tests/test_compositing.py

```diff
@@ -39,3 +39,28 @@ def test_clips_array_duration():
     video = clips_array([[red, green, blue]]).set_duration(5)
     video.write_videofile(join(TMP_DIR, "test_clips_array.mp4"))
     close_all_clips(locals())
+
+
+def test_blended_composite_video_clip():
+    bg = VideoFileClip("media/video_for_blend_modes.mp4").set_duration(1.0)
+    text_kwargs = {"font": FONT, "fontsize": 96, "color": "#bbbb11"}
+    t1 = (
+        TextClip("SOFT BLEND", **text_kwargs)
+        .set_duration(1.0)
+        .set_position(("center", "top"))
+    )
+    t2 = (
+        TextClip("HARD BLEND", **text_kwargs)
+        .set_duration(1.0)
+        .set_position(("center", "bottom"))
+    )
+    comp = BlendedCompositeVideoClip(
+        [bg, t1, t2],
+        clips_blending=[
+            {"blend_mode": "normal"},
+            {"blend_mode": "soft_light", "blend_opacity": 0.8},
+            {"blend_mode": "hard_light", "blend_weight": 0.8},
+        ],
+    )
+    comp.write_videofile(join(TMP_DIR, "test_blended_composition.mp4"))
+    return
```

Comment:
how come there are no assertion(s) on this test?

Reply:
Well it is really just to check that it does not crash. I don't know how you'd actually be able to check the results given how many pixels there are. I suppose that I could generate a clip that I check now, and then compare that static clip with the generated clip each time. Would that seem like a better idea worth doing? I suppose that would also require that `VideoFileClip` for loading the 'correct' clip is deterministic enough, which I'm not sure that it is.

Reply:
cool. that seems like a legit reason. maybe can indicate in the comments or something